lib/server/controllers/logs.controller.js

```diff
@@ -320,6 +315,27 @@ const getRootLogsHandler = async (request, response) => {
     }
 };
 
+// eslint-disable-next-line valid-jsdoc
+/**
+ * Get all child logs by parent id
+ */
+const getChildLogsByParentIdHandler = async (request, response) => {
+    const requestDto = await dtoValidator(GetLogDto, request, response);
```

Comment:
I apologize, I did not notice at first, but here you should not use the same DTO, because its purpose is not to get one log by id. 
Here you can directly generate the DTO on the fly using DtoFactory as you did well for root logs.
I would also rather name the parameter `parentLogId` instead of `logId`

Reply:
done